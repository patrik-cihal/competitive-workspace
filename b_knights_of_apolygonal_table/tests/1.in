7 2
2 4 6 7 8 9 10
10 8 4 8 4 5 9