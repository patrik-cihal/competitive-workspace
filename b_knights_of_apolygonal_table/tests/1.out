10 18 22 26 22 23 27 
